test/integration/resultsPage.js

```diff
@@ -40,10 +40,48 @@ describe('Results page', () => {
 describe('Results page error handling', () => {
   describe('when search is not included', () => {
     it('should return a descriptive error messages', (done) => {
+      const search = null;
+      const errorMessage = messages.emptySearchMessage();
```

Comment:
It's a matter of taste but I always prefer not using production code in my assertions. I'd prefer to to have the string explicitly stated here (e.g. "You didn't enter any text into the search box. You must type at least part of your GP surgery's name before searching.").

Doing so means I don't have to look anywhere else to see what the message actually is. Also if two errors combine (messages.emptySearchMessage is set to "" and error handling fails then the test won't incorrectly pass. Code duplication in this instance is a good thing.

I know others do not agree with this so feel free to accept, reject or discuss.